src/ApparentHorizons/Tags.hpp

```diff
@@ -366,6 +366,24 @@ struct RicciScalarCompute : RicciScalar, db::ComputeTag {
   using return_type = Scalar<DataVector>;
 };
 
+/// The pointwise maximum of the Strahlkorper's intrinsic Ricci scalar
+/// curvature.
+struct MaxRicciScalar : db::SimpleTag {
+  using type = double;
+};
+
+/// Computes the pointwise maximum of the Strahlkorper's intrinsic Ricci
+/// scalar curvature.
+template <typename Frame>
+struct MaxRicciScalarCompute : MaxRicciScalar, db::ComputeTag {
```

Comment:
This may be me forgetting how AH stuff works, but why is this templated on Frame but the simple tag is not?

Reply:
I'm not sure, I will consult with Dr. Lovelace and I will get back to you.

Reply:
Please try removing line 377 and let me know if you get a compiler error

Reply:
When I take the Frame out, I get a complaint from Test_tags.hpp, I'm not sure how to change the code to not take a Frame here. I get. 

error: expected '>'
      StrahlkorperTags::MaxRicciScalarCompute<Frame::Inertial>>(

It complains about the <Frame here though it is out of the code. 

This is what I have in Test_Tags.hpp 

`TestHelpers::db::test_compute_tag <
      StrahlkorperTags::MaxRicciScalarCompute("MaxRicciScalar");`

Reply:
You've removed one too many '>' characters.  You want
`TestHelpers::db::test_compute_tag<StrahlkorperTags::MaxRicciScalarCompute>("MaxRicciScalar");`.

Reply:
Thank you, @wthrowe , everything compiled fine, I'll move on to the test.